.github/workflows/publish.yaml

```diff
@@ -9,6 +9,9 @@ on:
       tag_name:
         required: true
         type: string
+    secrets:
+      publish_token:
```

Comment:
Does this need to match the case of the secret, which is `PUBLISH_TOKEN`?

Reply:
apparently not, I did test it by triggering a release workflow at this commit: https://github.com/aspect-build/rules_lint/actions/runs/14095611671